### PR TITLE
chore(pingcap/tidb): update trigger for CI job `pull-br-integration-test`

### LINKED ISSUE #3039: also trigger`BR` integration testing when changes are made to `DDL` module. @BornChanger @Benjamin2037

### PR DESCRIPTION
**Pull request by wuhuizuo**
Also trigger`BR` integration testing when changes are made to `DDL` module. @BornChanger @Benjamin2037

Fixes #3039

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: prow-jobs/pingcap/tidb/latest-presubmits.yaml [hunk 1]
```diff
@@ -102,7 +102,7 @@ presubmits:
       agent: jenkins
       decorate: false # need add this.
       context: pull-br-integration-test
-      run_if_changed: "br/.*"
+      run_if_changed: "(br|pkg/ddl)/.*"
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/tidb/latest-presubmits.yaml:105]: does it need to cover `pkg/domain`?

**Comment by wuhuizuo** [comment]
/cc @BornChanger @Benjamin2037 @D3Hunter
PTAL

File: prow-jobs/pingcap/tidb/latest-presubmits.yaml (hunk 1 above)
Comment (D3Hunter) [reply prow-jobs/pingcap/tidb/latest-presubmits.yaml:105]: yes

Comment (wuhuizuo) [inline prow-jobs/pingcap/tidb/latest-presubmits.yaml:105]:
```suggestion
      run_if_changed: "(br|pkg/ddl|pkg/domain)/.*"
```

**Pull request merged**